### PR TITLE
Add interpreter to GA

### PR DESCRIPTION
**Pull request by HardNorth**

### CODE REVIEW

**Comment by HardNorth** [comment]
@iivanou Please ignore codecov check for now. It will work better in further.

File: reportportal_client/external/google_analytics.pyi [hunk 1]
```diff
@@ -7,4 +7,6 @@ logger: Logger
 
 def _get_client_info() -> tuple: ...
 
+def _get_platform_info() -> str: ...
```
Comment (iivanou) [inline reportportal_client/external/google_analytics.pyi:10]: Use `Text` type for compatibility with Python 2.

File: reportportal_client/external/google_analytics.py [hunk 2]
```diff
@@ -34,6 +35,14 @@ def _get_client_info():
     return client.project_name, client.version
 
 
+def _get_platform_info():
+    """Get current platform basic info, e.g.: 'Python 3.6.1'.
+
+    :return: str represents the current platform, e.g.: 'Python 3.6.1'
+    """
+    return "Python " + python_version()
```
Comment (iivanou) [inline reportportal_client/external/google_analytics.py:43]: Let's use either single quotes or doubles within the same module.

File: tests/test_analytics.py [hunk 3]
```diff
@@ -22,11 +22,15 @@
 from reportportal_client.external.constants import GA_ENDPOINT, GA_INSTANCE
 from reportportal_client.external.google_analytics import send_event
 
+TEST_PYTHON_VERSION = '3.6.6'
```
Comment (iivanou) [inline tests/test_analytics.py:25]: No need to keep this variable. Just use `return_value=3.6.6` when patching the version.

**Review by iivanou — APPROVED** [review]

**Pull request merged**